policies/recipes/steps.go

```diff
@@ -370,51 +372,60 @@ func extractTar(tarName string, dst string, archiveType osconfigpb.SoftwareRecip
 }
 
 // StepMsiInstallation builds the command for a MsiInstallation step
-func StepMsiInstallation(step *osconfigpb.SoftwareRecipe_Step_MsiInstallation, artifacts map[string]string) error {
-	fmt.Println("StepMsiInstallation")
-	return nil
+func StepMsiInstallation(step *osconfigpb.SoftwareRecipe_Step_InstallMsi, artifacts map[string]string, runEnvs []string, stepDir string) error {
+	artifact := step.GetArtifactId()
+	path, ok := artifacts[artifact]
+	if !ok {
+		return fmt.Errorf("%q not found in artifact map", artifact)
+	}
+	args := step.Flags
+	if len(args) == 0 {
+		args = []string{"/i"}
+	}
+	args = append(args, path)
```

Comment:
add these args = /qn, /norestart
and these return codes = 0, 1641, 3010 (even though we say /norestart)